src/components/panes/declared-variable.jsx

```diff
@@ -5,15 +5,15 @@ import { ValueRenderer } from '../reps/value-renderer'
 
 export class DeclaredVariable extends React.Component {
   static propTypes = {
-    varName: PropTypes.string,
+    value: PropTypes.string,
```

Comment:
why did you remove the varName proptype? we should check that as well s the value, no?

Reply:
I removed the `varName` because this makes the `DeclaredVariable` more flexible to deal with other values, not just stuff that's in `window`, but happy to change this back / make a different component.

Reply:
i think what you did re `window` makes sense, we just want the proptypes check for both props --
`varName` and `values` are both passed in as props, no? (varName is still used on line 14: `<div className="declared-variable-name">{this.props.varName} = </div>`)

Reply:
yep good call :)